lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.html

```diff
@@ -23,8 +23,10 @@
               (closed)="onClosed()">
         <div #filterContainer
               role="menuitem"
+             tabindex="0"
              (keydown.tab)="$event.stopPropagation();">
             <div (click)="$event.stopPropagation()"
+                 tabindex="0" (keyup.enter)="$event.stopPropagation()"
```

Comment:
could you move (keyup.enter)="$event.stopPropagation()" to next line to keep consistency (you can see that every attribute for this tag is one under other)